File: Java/missingNumber.java
import java.util.Arrays;
class Solution {
    public int missingNumber(int[] nums) {

        Arrays.sort(nums);

        int value = 0;
        int missing = 0;

        for (int i = 0; i < nums.length; i++) {
            if (nums[i] == value) {
                value++;
            }
        }
        return value;
    }
}

File: Easy/TypeScript/missingNumber.ts
function missingNumber(nums: number[]): number {

    // Javascript sorts alphabetically so have to 
    // specific value
    nums.sort((a,b) => a - b);
    let missing = 0;
    
    for (let i = 0; i < nums.length; i++) {
        
        if (nums[i] === missing) {
            missing++;
        } 
    }
    
    return missing;
};
